### ISSUE TITLE
Should allow multiple JSX nodes within Then and Else

### ISSUE THREAD

**Issue by taarimalta** [romac/react-if#13]
Thanks for the wonderful library.

Can we please allow constructs such as

```
<Then>
<div>Content 1</div>
<div>Content 2</div>
</Then>
```

in both Then and Else

Instead of having to do the following

```
<Then>
<span>
<div>Content 1</div>
<div>Content 2</div>
</span>
</Then>
```

**Comment by romac** [comment]
Indeed, that would be nice! Sadly, I cannot take care of it at the moment, but would gladly accept a pull request :)

**Comment by vivcogit** [comment]
I created pull request for solving this problem - #20

**Comment by davidcalhoun** [comment]
Note that as of React `16.2.0` you can now use Fragments without worrying about creating extraneous DOM nodes:

```jsx
import React, { Fragment } from 'react';

...

<Then>
    <Fragment>
        <div>Content 1</div>
        <div>Content 2</div>
    </Fragment>
</Then>
```

**Comment by meszaros-lajos-gyorgy** [comment]
The PropTypes definition should be changed for children to PropTypes.node. I'll create a PR in a few minutes for that.

**Comment by romac** [comment]
Fixed in #31 and released in v3.0.0
